test/unit/remote-config/remote-config.spec.ts

```diff
@@ -295,6 +300,9 @@ describe('RemoteConfig', () => {
           'message', `Remote Config conditions must be an array`);
     });
 
+    // validate input template
+    testInputTemplate((t: RemoteConfigTemplate) => { remoteConfig.validateTemplate(t); });
```

Comment:
Can't you just do `testInputTemplate(remoteConfig.validateTemplate);`

Reply:
I tried that but it gives me the following type error. Not quite sure why.
```
    testInvalidInputTemplates(remoteConfig.validateTemplate);
                                           ^
TypeError: Cannot read property 'validateTemplate' of undefined
```